lib/src/clients.dart

```diff
@@ -279,33 +278,33 @@ class MultipartMediaUploader {
     // This guarantees us that [_body] cannot contain a valid multipart
     // boundary.
     var bodyHead = '--$_boundary\r\n'
-        "Content-Type: $CONTENT_TYPE_JSON_UTF8\r\n\r\n" +
+        'Content-Type: $CONTENT_TYPE_JSON_UTF8\r\n\r\n' +
         _body +
         '\r\n--$_boundary\r\n'
-        "Content-Type: ${_uploadMedia.contentType}\r\n"
-        "Content-Transfer-Encoding: base64\r\n\r\n";
+        'Content-Type: ${_uploadMedia.contentType}\r\n'
+        'Content-Transfer-Encoding: base64\r\n\r\n';
     var bodyTail = '\r\n--$_boundary--';
 
     var totalLength =
         bodyHead.length + base64MediaStreamLength + bodyTail.length;
 
-    var bodyController = new StreamController<List<int>>();
+    var bodyController = StreamController<List<int>>();
     bodyController.add(utf8.encode(bodyHead));
     bodyController.addStream(base64MediaStream).then((_) {
       bodyController.add(utf8.encode(bodyTail));
-    }).catchError((error, stack) {
+    }).catchError((error, StackTrace stack) {
       bodyController.addError(error, stack);
     }).then((_) {
       bodyController.close();
     });
 
     var headers = {
       'user-agent': _userAgent,
-      'content-type': "multipart/related; boundary=\"$_boundary\"",
+      'content-type': 'multipart/related; boundary=\"$_boundary\"',
```

Comment:
No need to escape " anymore.

Reply:
Not going to mess with this, but understood. We need better auto-fix tools here.